Formula/libgrape-lite.rb

```diff
@@ -0,0 +1,62 @@
+class LibgrapeLite < Formula
+  desc "C++ library for parallel graph processing"
+  homepage "https://github.com/alibaba/libgrape-lite"
+  url "https://github.com/alibaba/libgrape-lite/archive/refs/tags/v0.1.0.tar.gz"
+  sha256 "aafc8e5e9b122c4bde4370ee3f2ab8afb043f2a1547cacc062991ad420e3b9a4"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+
+  depends_on "glog"
+  depends_on "open-mpi"
```

Comment:
Does it not work with the macOS version?

Reply:
It does work on macOS. `open-mpi` is available on Mac in `brew install` as well. Could you please explain a bit more about the concern about macOS here?

Reply:
As far as I remember macOS comes with an open-mpi alternative. Can we not use that?

Reply:
I haven't found a builtin MPI on my Mac, but I found the following statement from the open-mpi FAQ: https://www.open-mpi.org/faq/?category=osx

> Open MPI v1.2.3 was included in some older versions of OS X, starting with version 10.5 (Leopard). It was removed in more recent versions of OS X (we're not sure in which version it disappeared — *but your best bet is to simply download a modern version of Open MPI for your modern version of OS X*).
> 
> Note, however, that OS X Leopard does not include a Fortran compiler, so the OS X-shipped version of Open MPI does not include Fortran support.
> 
> If you need/want Fortran support, you will need to build your own copy of Open MPI (assumedly when you have a Fortran compiler installed). The Open MPI team strongly recommends not overwriting the OS X-installed version of Open MPI, but rather installing it somewhere else (e.g., /opt/openmpi).


I think it was removed in recently versions of Mac OS (may as early as 10.15, the oldest version we are running the CI). So I think it should be OK to install the "open-mpi" package using the homebrew as the dependency ?

Reply:
I guess I was wrong then.